docs/src/user_guide/quick_start_guide.md

```diff
@@ -1,3 +1,59 @@
 # Quick Start Guide
 
 This section is still under development. Please visit [PowerSystems.jl Intro](https://nbviewer.jupyter.org/github/NREL-SIIP/SIIPExamples.jl/blob/master/notebook/PowerSystems_examples/PowerSystems_intro.ipynb) in the mean time.
+
+## Definitions to work with Time Series Data
```

Comment:
This is good content. Would it make more sense to move it to the other page?  To me a quick start guide is something that shows you how to create a system and get running.

Reply:
I am torn about that, I feel this is a very basic part. I say we leave it here for now and in the PR for docs, we iron out the details

Reply:
I'll state my main comment in a different way.  I think a quick start guide should have a list of steps to get running, not paragraphs of text and definitions.

Also, the table doesn't render in GitHub Markdown.
https://github.com/NREL-SIIP/PowerSystems.jl/blob/1912306df97edea6cfd902d4dd79ee9bd36d7240/docs/src/user_guide/quick_start_guide.md